src/js/hca/validation.js

```diff
@@ -3,14 +3,27 @@ import _ from 'lodash/fp';
 import { convertToDateField, validateCurrentOrPastDate } from '../common/schemaform/validation';
 import { isValidDateRange } from '../common/utils/validations';
 
+function calculateEndDate() {
+  let endDateLimit;
+  if (process.env.BUILDTYPE === 'production') {
```

Comment:
We normally use `__BUILDTYPE__` instead of `process.env.BUILDTYPE`

Reply:
ok